docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: '3.7'
+version: '3.1'
```

Comment:
It's not clear to me whether we can set `docker-compose.yml` to version 3.1 and the others to version 3.2.  When this code is deployed we create a `docker-compose.override.yml` symlink to one of the `docker-compose.*.yml` files, and that content is automatically munged with the output of `docker-compose.yml`.

When I create the symlink and run `docker-compose config` the output indicates that the whole shebang gets version 3.1, which won't work.

Reply:
Ah, I overlooked that, and I believe you are correct. I updated and added a comment to explain in d63ce80.